targettests/braket/bug_qubit.cpp

```diff
@@ -0,0 +1,48 @@
+/*******************************************************************************
```

Comment:
For `targettests/braket`  instead of making copies, create symbolic link to the corresponding files in `targettests/execution`